prestodb/cdh5-base/Dockerfile

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM teradatalabs/centos6-java8-oracle:unlabelled
-MAINTAINER Teradata Docker Team <docker@teradata.com>
+FROM prestodb/centos6-oj8:unlabelled
```

Comment:
Don't you need to add `flag/cdh-base.flags` file here and for other images as well?

Reply:
Nevermind this commit. These files are created dynamically.